packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -14,19 +34,121 @@ defineProps<OnyxTableProps>();
 @use "../../styles/mixins/density.scss";
 @use "../../styles/mixins/layers";
 
+/**
+* Defines all border styles for the table.
+*/
+@mixin define-borders() {
+  $border: var(--onyx-1px-in-rem) solid var(--onyx-color-base-neutral-300);
+  $border-radius: var(--onyx-radius-sm);
+
+  border-spacing: 0;
+  border-collapse: separate;
+  border-radius: $border-radius;
+
+  // border styles
+  th,
+  td {
+    border-bottom: $border;
+
+    &:first-child {
+      border-left: $border;
+    }
+    &:last-child {
+      border-right: $border;
+    }
+  }
+
+  th {
+    border-top: $border;
+  }
+
+  // border radius
+  tr:first-child th:first-child {
+    border-top-left-radius: $border-radius;
+  }
+
+  tr:first-child th:last-child {
+    border-top-right-radius: $border-radius;
+  }
+
+  tr:last-child td:first-child {
```

Comment:
do the border bottoms still work if we have no td rows? I mean only the header, when e.g. the table content is still loading / is empty initially. 

well, probably we'll look after that when implementing loading/empty behavior 😋

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/67898185/226ed22e-68cb-4afd-880e-be8acefdf299)

No the bottom borders will not be rounded then but I assumed that the table always has either content or an "empty/loading" state that we will implement in the future.

But if you want I can take care of this edge case

Reply:
let's merge it for now and ask jannick tomorrow if we need a follow up ticket for that :>